src/rednoise.c

```diff
@@ -62,16 +70,69 @@ int main(int argc, char *argv[])
         float *powbuf, powargr, powargi;
         fcomplex *newbuf, *oldbuf, *inbuf1, *inbuf2, *outbuf, *tempzz;
 
+        struct stat info;
+        
         /* Read the info file, and write a copy */
         {
             char *newinf;
+            char *infdir;
+            char *filenm;
             infodata idata;
+
+
             readinf(&idata, rootfilenm);
             numsamp = idata.N;
             T = numsamp * idata.dt;
+
+            split_path_file(idata.name, &infdir, &filenm);
+            
             newinf = (char *) calloc(strlen(idata.name) + 5, sizeof(char));
+
+            /* Uncomment for debugging 
+            printf("idata.name: %s\n", idata.name);
+            printf("rootfilenm: %s\n", rootfilenm);
+            printf("infdir: %s\n", infdir);
+            printf("filenm: %s\n", filenm)
+            */
+
+            if( stat( infdir, &info ) != 0 ) {
```

Comment:
For indentation, I run all of the C code through the `indent` command using the following command line:  `indent -kr -nut -l85 FILE.c`

Reply:
Great, I did not know that command. I will give it a try.